Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -906,43 +897,30 @@ void CSequenceBatchRenderDialog::CaptureItemStart()
 
     CV_TrackViewRenderOutputCapturing = 1;
 
-    SRenderItem renderItem = m_renderItems[m_renderContext.currentItemIndex];
-    IAnimSequence* nextSequence = renderItem.pSequence;
-    // Initialize the next one for the batch rendering.
-    // Set the active shot.
-    m_renderContext.pActiveDirectorBU = nextSequence->GetActiveDirector();
-    nextSequence->SetActiveDirector(renderItem.pDirectorNode);
-
-    // Back up flags and range of the sequence.
-    m_renderContext.flagBU = nextSequence->GetFlags();
-    m_renderContext.rangeBU = nextSequence->GetTimeRange();
-
-    // Change flags and range of the sequence so that it automatically starts
-    // once the game mode kicks in with the specified range.
-    nextSequence->SetFlags(m_renderContext.flagBU | IAnimSequence::eSeqFlags_PlayOnReset);
+    const SRenderItem& renderItem = m_renderItems[m_renderContext.currentItemIndex];
 
     m_renderContext.captureOptions.timeStep = 1.0f / renderItem.fps;
-
-    Range newRange = renderItem.frameRange;
-    newRange.start -= m_renderContext.captureOptions.timeStep;
-    renderItem.pSequence->SetTimeRange(newRange);
-
+  
     // Set up the custom config cvars for this item.
-    for (size_t i = 0; i < renderItem.cvars.size(); ++i)
+    IConsole* pConsole = GetIEditor()->GetSystem()->GetIConsole();
+    for (const auto& cvar : renderItem.cvars)
     {
-        GetIEditor()->GetSystem()->GetIConsole()->ExecuteString(renderItem.cvars[static_cast<int>(i)].toUtf8().data());
+        pConsole->ExecuteString(cvar.toUtf8().data());
     }
 
     // Set specific capture options for this item.
     m_renderContext.captureOptions.prefix = renderItem.prefix.toUtf8().data();
 
-    Range rng = nextSequence->GetTimeRange();
-    m_renderContext.captureOptions.duration = rng.end - rng.start;
     QString folder = renderItem.folder;
-    QString itemText = m_ui->m_renderList->model()->index(m_renderContext.currentItemIndex, 0).data().toString();
-    itemText.replace('/', '-'); // A full sequence name can have slash characters which aren't suitable for a file name.
-    folder += "/";
-    folder += itemText;
+    folder.append('/');
+
+    static const QString reservedCharacters("/\\:*?\"<>|");
```

Comment:
Reserved characters as in cannot be used to name folders ? You might want to use AssetUtils SanitizeFileName or something like that